### DISCUSSION TITLE
Select verticies in a curve (drag select)?

### DISCUSSION THREAD

**Discussion by Hipshot-zz** [Garux/netradiant-custom#174] [Q&A] [answered]
Seems like such a basic function that I can't find, but how do I drag select to mark a bunch of verticies in a curve mesh in q3? I did look at the mouse shortcut guide, but I'm not sure which one of all these commands it is or if it's even in there? Or do I really need to select them one by one?

**Comment by Garux** [comment, marked as answer]
```
             shift + m2 drag        rectangular selector (select/deselect/toggle, complete depth, partial match)
```

**Comment by Hipshot-zz** [comment]
Oh ok, btw, it seems you can't drag from the top right, other directions works (at least this happens for me).

**Comment by Garux** [comment]
Can't drag or can't select? Because *drag from the top right* is deselecting mode (colorcoded by blue).

**Reply by Hipshot-zz** [reply]
Ah get it, thanks.
